solidity/test/DepositFundingTest.js

```diff
@@ -394,6 +393,58 @@ describe("DepositFunding", async function() {
     })
   })
 
+  describe("requestFunderAbort", async () => {
+    let timer
+    let owner
+
+    before(async () => {
+      timer = await tbtcConstants.getFundingTimeout.call()
+      owner = accounts[1]
+      await tbtcDepositToken.forceMint(
+        owner,
+        web3.utils.toBN(testDeposit.address),
+      )
+    })
+
+    beforeEach(async () => {
```

Comment:
nit, but we can move all of this to the `before` block, and then: 
```
    beforeEach(async () => {
      await createSnapshot()
    })

    afterEach(async () => {
      await restoreSnapshot()
    })
```

Reply:
Bah, I actually wanted to do this, but I think we're close enough here that I'm going to punt it to an imaginary future improvement.